dbt/adapters/bigquery/python_submissions.py

```diff
@@ -10,6 +10,10 @@
     from google.cloud import storage, dataproc_v1  # type: ignore
 except ImportError:
     _has_dataproc_lib = False
+    from unittest.mock import MagicMock
+
```

Comment:
This is for type hints only, we will raise an error if anything tries to actually run models since the `_has_dataproc_lib ` is set